react-components/src/components/Architecture/pointsOfInterest/usePoIDomainObject.ts

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import { PointsOfInterestDomainObject } from '../../../architecture';
+import { PointsOfInterestTool } from '../../../architecture/concrete/pointsOfInterest/PointsOfInterestTool';
+import { useRenderTarget } from '../../RevealCanvas';
+import { useOnUpdate } from '../useOnUpdate';
+
+export function usePoIDomainObject(): PointsOfInterestDomainObject<any> | undefined {
```

Comment:
is there a way to get rid of the `any` here?

Reply:
I'll write `unknown`